test/e2e/specs/OIDC_spec.js

```diff
@@ -57,6 +57,24 @@ describe('OIDC flows', function() {
       expect(oidcApp.getIdTokenUser()).toBe('Saml Jackson');
     });
 
+    it('throws form error if auth client returns with OAuth error', function() {
+      setup({
+        baseUrl: '{{{WIDGET_TEST_SERVER}}}',
+        clientId: 'rW47c465c1wc3MKzHznu',
+        redirectUri: 'http://localhost:3000/done',
+        authParams: {
+          responseType: 'id_token',
+          scope: ['openid', 'email', 'profile', 'address', 'phone']
+        }
+      });
+      primaryAuth.loginToForm('{{{WIDGET_BASIC_USER_5}}}', '{{{WIDGET_BASIC_PASSWORD_5}}}');
+      var element = primaryAuth.errorMessage();
```

Comment:
Let's go for this here:
```javascript
primaryAuth.loginToForm(stuff);
expect(primaryAuth.getErrorMessage()).toBe('User is not assigned etc');
```
And add `getErrorMessage` to the PrimaryAuth page, modeled after:
https://github.com/okta/okta-signin-widget/blob/master/test/e2e/page-objects/OIDCAppPage.js#L31

Where the main points are:
- We're putting the page specific logic in the page object
- We're using the protractor expected condition helpers